src/compound/MorphoStorage.sol

```diff
@@ -53,7 +52,7 @@ abstract contract MorphoStorage is OwnableUpgradeable, ReentrancyGuardUpgradeabl
     /// CONTRACTS AND ADDRESSES ///
 
     IPositionsManager public positionsManager;
-    IIncentivesVault public incentivesVault;
+    address public incentivesVault; // Deprecated.
```

Comment:
I would also remove the getter I thing
```suggestion
    address internal incentivesVault; // Deprecated.
```

Reply:
It would change the abi so I don't think we should do it